srcpkgs/zig/template

```diff
@@ -11,9 +11,57 @@ license="MIT"
 homepage="https://ziglang.org/"
 distfiles="https://ziglang.org/download/${version}/zig-${version}.tar.xz"
 checksum=5d167dc19354282dd35dd17b38e99e1763713b9be8a4ba9e9e69284e059e7204
+patch_args="-Np1"
 nopie=yes
-nocross=yes
+
+if [ "${CROSS_BUILD}" ]; then
+	# TODO _mcpu might be a candidate for cross-profiles
+	#
+	# Set _cross_triple and _mcpu for target triples that are not directly
+	# available in zig's targets.
+	#
+	# List targets with: zig targets
+
+	case "${XBPS_CROSS_TRIPLET}" in
+		armv7l*)
+			_cross_triple="arm-linux-${XBPS_CROSS_TRIPLET#arm*-linux-*}"
```

Comment:
Why is this necessary?

Reply:
Some of zigs targets don't exactly match up with gcc triplets. In this case, all of arm32 is under `arm-...` with the different subarchs identified via the mcpu parameter.

Reply:
Fair enough. I think the substitution I'm doing below is a bit neater, if you'd like to use that.

Reply:
cleaned this part up